appcues/src/main/java/com/appcues/debugger/ui/details/DebuggerEventDetails.kt

```diff
@@ -59,16 +59,11 @@ internal fun DebuggerEventDetails(debuggerEventItem: DebuggerEventItem?, onBackP
 
         details(debuggerEventItem)
 
-        if (debuggerEventItem.properties != null && debuggerEventItem.properties.isNotEmpty()) {
-            propertiesTitle()
-
-            properties(debuggerEventItem.properties)
-        }
-
-        if (debuggerEventItem.identityProperties != null && debuggerEventItem.identityProperties.isNotEmpty()) {
-            identityPropertiesTitle()
-
-            properties(debuggerEventItem.identityProperties)
+        debuggerEventItem.propertySections.forEach {
+            if (it.properties != null && it.properties.isNotEmpty()) {
+                propertiesTitle(it.title)
+                properties(it.properties)
+            }
```

Comment:
the debugger UI is now slightly simpler, just looping over the sections provided and using the title provided

Reply:
nice improv to the debugger UI